mods/hide-unc.wh.cpp

```diff
@@ -0,0 +1,346 @@
+            BOOL result = ExtTextOutW_Original(hdc, x, y, options, lprect, buffer, (UINT)wcslen(buffer), lpDx);
+            free(buffer);
+            return result;
+        }
+    }
+    return ExtTextOutW_Original(hdc, x, y, options, lprect, lpString, c, lpDx);
+}
+
+typedef BOOL (WINAPI *SetWindowTextW_t)(HWND hWnd, LPCWSTR lpString);
+SetWindowTextW_t SetWindowTextW_Original;
+
+BOOL WINAPI SetWindowTextW_Hook(HWND hWnd, LPCWSTR lpString) {
+    if (lpString && wcsstr(lpString, L" (\\\\")) {
+        size_t len = wcslen(lpString);
+        WCHAR* buffer = (WCHAR*)malloc((len + 1) * sizeof(WCHAR));
+        if (buffer) {
+            wcscpy_s(buffer, len + 1, lpString);
+            CleanUNCPath(buffer, len + 1);
+            BOOL result = SetWindowTextW_Original(hWnd, buffer);
+            free(buffer);
+            return result;
+        }
+    }
+    return SetWindowTextW_Original(hWnd, lpString);
+}
+
+WNDPROC g_OrigComboBoxProc = NULL;
+
+LRESULT CALLBACK ComboBoxSubclassProc(HWND hWnd, UINT uMsg, WPARAM wParam, LPARAM lParam) {
+    if (uMsg == CB_ADDSTRING || uMsg == CB_INSERTSTRING) {
+        LPWSTR text = (LPWSTR)lParam;
+        if (text && wcsstr(text, L" (\\\\")) {
+            size_t len = wcslen(text) + 1;
+            WCHAR* buffer = (WCHAR*)malloc(len * sizeof(WCHAR));
+            if (buffer) {
+                wcscpy_s(buffer, len, text);
+                CleanUNCPath(buffer, len);
+                LRESULT result = CallWindowProc(g_OrigComboBoxProc, hWnd, uMsg, wParam, (LPARAM)buffer);
+                free(buffer);
+                return result;
+            }
+        }
+    }
+    return CallWindowProc(g_OrigComboBoxProc, hWnd, uMsg, wParam, lParam);
+}
+
+typedef HWND (WINAPI *CreateWindowExW_t)(DWORD dwExStyle, LPCWSTR lpClassName, LPCWSTR lpWindowName, 
+    DWORD dwStyle, int X, int Y, int nWidth, int nHeight, HWND hWndParent, HMENU hMenu, HINSTANCE hInstance, LPVOID lpParam);
+CreateWindowExW_t CreateWindowExW_Original;
+
+HWND WINAPI CreateWindowExW_Hook(DWORD dwExStyle, LPCWSTR lpClassName, LPCWSTR lpWindowName, 
+    DWORD dwStyle, int X, int Y, int nWidth, int nHeight, HWND hWndParent, HMENU hMenu, HINSTANCE hInstance, LPVOID lpParam) {
+    
+    HWND hWnd = CreateWindowExW_Original(dwExStyle, lpClassName, lpWindowName, dwStyle, X, Y, nWidth, nHeight, hWndParent, hMenu, hInstance, lpParam);
+    
+    if (hWnd && lpClassName) {
+        WCHAR className[256] = {0};
+        if (IS_INTRESOURCE(lpClassName)) {
+            GetClassNameW(hWnd, className, 256);
+        } else {
+            wcscpy_s(className, 256, lpClassName);
+        }
+        
+        if (wcscmp(className, L"ComboBox") == 0 && hWndParent) {
+            WCHAR parentClass[256];
+            GetClassNameW(hWndParent, parentClass, 256);
+            if (wcsstr(parentClass, L"#32770")) {
+                WNDPROC oldProc = (WNDPROC)SetWindowLongPtr(hWnd, GWLP_WNDPROC, (LONG_PTR)ComboBoxSubclassProc);
```

Comment:
This also isn't being restored, right? I suggest using `WindhawkUtils::SetWindowSubclassFromAnyThread`, remember which windows are subclassed, and remove the subclass on uninit. You can look at existing mods for examples.